test/metabase/api/user_test.clj

```diff
@@ -66,9 +66,10 @@
         (let [result (->> ((mt/user-http-request :crowberto :get 200 "user") :data)
                           (filter mt/test-user?))]
           ;; since this is an admin, all keys are available on each user
-          (is (= (set (concat
-                       user/admin-or-self-visible-columns
-                       [:common_name :group_ids :personal_collection_id]))
+          (is (= (disj (set (concat
+                             user/admin-or-self-visible-columns
+                             [:common_name :group_ids :personal_collection_id]))
+                       :type)
```

Comment:
Would it be better to just exclude `:type` from `default-user-columns` if we're not typically going to return it when fetching users? removing `:type` manually here feels a little error-prone

Reply:
We needed to have it in `default-user-columns` so that the after-select hook would work - but now that we don't need to do any special massaging of the user's name, we can remove this.